src/index.tsx

```diff
@@ -1,4 +1,6 @@
 // Dependencies
+import * as React from "react";
```

Comment:
Is there anything in this file that needs react?

Reply:
Good catch, I must have missed removing that.